clustermesh-apiserver/ciliumendpointslice/ciliumendpointslice.go

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Nit. I'd maybe move this directly to `clustermesh-apiserver/clustermesh/k8s` for consistency.